versioned_docs/version-2.0.0/quickstart/go-mux-sql.md

```diff
@@ -433,16 +423,8 @@ Depending on your OS, choose your adventure:
   docker volume create --driver local --opt type=debugfs --opt device=debugfs debugfs
   ```
 
-  ```bash
-  alias keploy='sudo docker run --pull always --name keploy-v2 -p 16789:16789 --privileged --pid=host -it -v $(pwd):$(pwd) -w $(pwd) -v /sys/fs/cgroup:/sys/fs/cgroup -v debugfs:/sys/kernel/debug:rw -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock -v '"$HOME"'/.keploy-config:/root/.keploy-config -v '"$HOME"'/.keploy:/root/.keploy --rm ghcr.io/keploy/keploy'
-  ```
-
   ### Use Keploy with Colima
 
-  ```bash
```

Comment:
After heading `use keploy with colima`, you can mention `colima start` in the md files.